cves/CVE-2007-5000.yml

```diff
@@ -103,8 +96,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
+  answer: |
+    It was reported to the security team that XSS attacks were possible on websites
+    where mod_imagemap was enabled.
```

Comment:
Do you know where this was reported to the security team? Like through a bug reporting tool, on forums, or some other way?

Reply:
I found some information on it on Red Hat Bugzilla. I'll add what I found.